types/index.d.ts

```diff
@@ -32,4 +40,17 @@ declare function xastscript(
   ...children: Children[]
 ): Element
 
+/**
+ * Create XML trees in xast.
+ *
+ * @param name Qualified name. Case sensitive and can contain a namespace prefix (such as rdf:RDF).
+ * @param attributes Map of attributes. Nullish (null or undefined) or NaN values are ignored, other values are turned to strings.
+ * @param children (Lists of) child nodes. When strings are encountered, they are mapped to Text nodes.
+ */
+declare function xastscript(
+  name: null,
+  attributes?: Attributes,
+  ...children: Children[]
+): Root
```

Comment:
Nice! I think this one can be removed though, as `attributes` are nonsensical / can’t be used on roots 🤔

Reply:
good catch updated in 7141b74